utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -672,6 +676,25 @@ int main(int argc, char **argv)
 
                     break;
                 }
+                case 11:
+                {
+                    testCaseName = "rain";
+
+                    Rpp32f rainPercentage = 7;
+                    Rpp32u rainHeight = 6;
+                    Rpp32u rainWidth = 1;
+                    Rpp32s slant = 0;
+                    for(int i = 0; i < batchSize; i++)
```

Comment:
Please add space after for
```suggestion
                    for (int i = 0; i < batchSize; i++)
```

Reply:
Done